docs/tasks/issuers/setup-acme/dns01/index.rst

```diff
@@ -64,6 +64,26 @@ Example usage::
 
 .. _supported-dns01-providers:
 
+Delegated Domains for DNS01
+===========================
+
+By default, cert-manager will not follow CNAME records pointing to subdomains.
+
+If granting cert-manager access to the root DNS zone is not desired, then the
+_acme-challenge.example.com subdomain can instead be delegated to some other,
+less privileged domain.
+This is done by adding an additional entry in `solvers`:
```

Comment:
It'd be great if you can clarify that the user will need to configure a CNAME record pointing to the domain that should be used to solve the challenge, as well as noting that they will need to provide DNS configuration/credentials for the zone that *should be updated* instead of the main one